apps/builder/app/canvas/shared/use-drag-drop.ts

```diff
@@ -242,6 +244,10 @@ export const useDragAndDrop = () => {
     },
 
     onStart({ data: dragInstanceSelector }) {
+      if (isInstanceDetachable(dragInstanceSelector) === false) {
+        toast.error("Cannot drag instance without its parent instance");
```

Comment:
need to check wording with @taylornowotny

Reply:
More semantic suggestion:
"Instance cannot be moved outside of its parent component."

This covers the fact that it can't be copied or deleted, not limited to drag.

Question - are we disabling drag entirely or just drag outside of the parent? User may still want to move things around within the parent container.

Reply:
I would figure this out once we have a use case. For now it is always used for cases where the order does not matter.